import-page/README.md

```diff
@@ -0,0 +1,59 @@
+# Import-Page
+
+Import-Page is a Lucid application that re-organizes shapes and elements on your documents into circles.
+
+## Getting Started
+
+To run your application locally, you will need to install the dependencies and run the development server. Additional instructions can be found [here](https://developer.lucid.co/extension-api/#debug-your-editor-extension)
+
+```bash
+/import-page/editorextensions/import-page$ npm install
+/import-page$ npx lucid-package test-editor-extension import-page
+```
+
+## Packaging for the Lucid Developer Portal
+
+Follow [these directions](https://developer.lucid.co/extension-api/#bundle-your-package-for-upload) to bundle, upload, and install your extension on your personal Lucid account.
+
+Note: the extensions for an app are managed under the "Packages" menu inside the application settings.
+Click the "+ New Version" button to upload your bundled package.
+
+## Folder / file structure
+
+The code is organized into the following folders:
+
+```
+└── import-page
+    └── editorextensions
+        └── import-page
+            └── src
+                └── extension.ts
+```
+
+The only relevant files are `extension.ts`, `importmodel.ts`, and `import.html`, which contain the code for the extension.
```

Comment:
It looks like `import.html` is in the resources folder separate from the other two. Consider making that clear in your folder structure.